build-config/scripts/apply-patch-series

```diff
@@ -18,7 +19,7 @@ PATCH_ROOT=`dirname $SERIES`
 cd "$SRC" &&
 git init && 
 git add . > /dev/null &&
-git commit -m "Unmodified $(basename $SRC) src" > /dev/null &&
+git commit -m "Unmodified $(basename $SRC) src" --allow-empty > /dev/null &&
```

Comment:
if we directly clone a repository using git clone then "git add ." will not add any file in the commit and git commit will fail with an error. which is why this is needed to allow an empty commit before applying patches from onie/patches